CHANGELOG.md

```diff
@@ -3,7 +3,8 @@
 ## unreleased
 
 * BraintreeCore
-  * Add 'appLinkReturnUri' to `BraintreeClient` constructors for Android App Link support
+  * Add `appLinkReturnUri` to `BraintreeClient` constructors for Android App Link support
```

Comment:
```suggestion
  * Add `appLinkReturnUri` to `BraintreeClient` constructors for Android App Link support (for PayPal web flows only)
```

Or something like that to make it clear that this can only be used with PayPal currently?